src/ansys/additive/core/parametric_study/parametric_runner.py

```diff
@@ -102,12 +102,12 @@ def simulate(
             simulation_ids_list = list()
             for sim_id in simulation_ids:
                 if sim_id not in view[ColumnNames.ID].values:
-                    LOG.warning(f"Simulation ID '{sim_id}' not found in the parametric study")
+                    LOG.debug(f"Simulation ID '{sim_id}' not found in the parametric study")
```

Comment:
This should be `LOG.warning()`. It was only line 107 that should be removed or debug.
```suggestion
                    LOG.warning(f"Simulation ID '{sim_id}' not found in the parametric study")
```